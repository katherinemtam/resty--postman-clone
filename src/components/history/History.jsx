import React from 'react';
import PropTypes from 'prop-types';

const History = ({ search, method }) => (
  <>
    <p>{method}</p>
    <p>{search}</p>
  </>
);

History.propTypes = {
  search: PropTypes.string.isRequired,
  method: PropTypes.string.isRequired,
};

export default History;

