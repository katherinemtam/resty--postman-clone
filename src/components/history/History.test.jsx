import React from 'react';
import { render, cleanup } from '@testing-library/react';
import History from './History';

describe('History component', () => {
  afterEach(() => cleanup());
  it('renders History', () => {
    const { asFragment } = render(<History />);
    expect(asFragment()).toMatchSnapshot();
  });
});
