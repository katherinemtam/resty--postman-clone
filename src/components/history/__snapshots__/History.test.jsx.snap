// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`History component renders History 1`] = `
<DocumentFragment>
  <article>
    <h4 />
    <p />
  </article>
</DocumentFragment>
`;
